test/e2e/storage/vsphere/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,27 @@
+package bootstrap
```

Comment:
You need to add license header to these new files, else bazel check will fail.

Reply:
Done.